src/Network/Receive/kRO/Sakexe_0.pm

```diff
@@ -431,7 +431,7 @@ sub new {
 		'02B8' => ['party_show_picker', 'a4 v C3 a8 v C', [qw(sourceID nameID identified broken upgrade cards location type)]], # 22
 		'02B9' => ['hotkeys'], # 191 # hotkeys:27
 		'02BB' => ['equipitem_damaged', 'v a4', [qw(slot ID)]], # 8
-		'02C1' => ['main_chat', 'v a4 a4 a*', [qw(len accountID color message)]], # -1		
+		'02C1' => ['npc_chat', 'x2 a4 a4 Z*', [qw(ID color message)]],
```

Comment:
you can keep len

v length is esqual to x2

'02C1' => ['npc_chat', 'v a4 a4 Z*', [qw(len ID color message)]],

Reply:
FIXED : 8844ae6